src/devices/Tcs3472x/TCS3472Type.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Tcs3472x
+{
+    public enum TCS3472Type
+    {
+        TCS34721 = 0x44,
+        TCS34725 = 0x44,
```

Comment:
values are doubled (here and below) - is this intentional?

Reply:
Yes, it is intentional. Some sensors have the same signature